### PR TITLE
Update runc vendor version to match Dockerfile's

### PR DESCRIPTION
**Pull request by mlaventure**
Update runc vendor revision to match the one present in the Dockerfile (and include missing fixes from libcontainer routine invoked directly by docker)

### CODE REVIEW

**Comment by thaJeztah** [comment]
ping @crosbymichael @tonistiigi please help reviewing this change. Could use some :eyes: on this

**Comment by LK4D4** [comment]
CI is failed
review is done

**Comment by crosbymichael** [comment]
lol

**Comment by crosbymichael** [comment]
Did the vendor script not remove the unused files/packages?  These are not new changes that were added.

**Comment by mlaventure** [comment]
initially the vendor script event removed some files from under the libnetwork vendoring which I didn't include here.

I'll check what's wrong with the tests.

**Comment by mlaventure** [comment]
@LK4D4 @crosbymichael @thaJeztah @tonistiigi 

- Fixed the tests
- Rerun the vendor script, and this time clean seemed to have worked correctly. A must better changeset :)

File: VERSION [hunk 1]
```diff
@@ -1 +1 @@
-1.11.1
+1.11.2
```
Comment (crosbymichael) [inline VERSION]: no, bad, bad

**Comment by thaJeztah** [comment]
Looking much better!

File: VERSION (hunk 1 above)
Comment (mlaventure) [inline VERSION]: why, why, why? It's a different commit, not a result of the vendoring.

Comment (crosbymichael) [inline VERSION]: why are you changing the version in this PR though?

Comment (mlaventure) [inline VERSION]: Two birds in one stone?

I can make a separate PR for that 1 char change if you prefer though.

Comment (crosbymichael) [inline VERSION]: the bump commit should always be the top commit for the release that gets tagged so as long as that happens and its top commit do as you wish.

Comment (mlaventure) [inline VERSION]: Looks like we've been doing it at bump branch creation now :)

Comment (thaJeztah) [inline VERSION]: I agree that having it in a separate PR is better

Comment (mlaventure) [inline VERSION]: Ok, making a new PR and updating this one.

Note, that, the bump version PR would have to be merged first.

Comment (mlaventure) [inline VERSION]: New PR for bump: https://github.com/docker/docker/pull/23029

**Comment by crosbymichael** [comment]
LGTM

**Comment by LK4D4** [comment]
LGTM

**Pull request merged**